src/components/node-list/styles/_filter.scss

```diff
@@ -68,12 +55,11 @@
   .pipeline-nodelist__row:hover
```

Comment:
The background highlight when hovering on sidebar rows seems to be missing on both themes now, might be an issue?

Reply:
Looks like I used the wrong syntax in _row.scss on lines 11 & 17. clearly i was tired when i did this last batch 🙈. fixed.